src/advanced/time.md

```diff
@@ -141,13 +141,10 @@ None.
 
 #### Response
 
-Example of JSON response:
+Example of response:
```

Comment:
Would you mind adding that the response is combined date and time in UTC as per [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)?